bindings/java/c/evmc-vm.c

```diff
@@ -88,6 +88,10 @@ JNIEXPORT void JNICALL Java_org_ethereum_evmc_EvmcVm_execute(JNIEnv* jenv,
                                                              jint jcode_size,
                                                              jobject jresult)
 {
+    jint rs = evmc_java_set_jvm(jenv);
```

Comment:
This is part of another PR. We should actually pass down the `jenv` via the context.